### PR TITLE
Sar 272/execution tests and contract

### PR DESCRIPTION
**Pull request by eawest2**
# Related Tickets

- [SAR-272](https://amida.atlassian.net/browse/SAR-272?atlOrigin=eyJpIjoiZGEyYTY2MWI2ZDIyNGI1ZTkxMDkwMTZkZWVjZjI0NmEiLCJwIjoiaiJ9)

# How Things Worked (or Didn't) Before This PR

There was no data contract, nor any tests testing whether the output was as expected.

# How Things Work Now (And How to Test)

Now there's a data contract in `./contract` and examples in `./contract/examples` and tests that work. 

run `yarn test` after having set up the Repo. You apparently need a private folder with the cql data, if your repo isn't set up this way already, ask James.

### CODE REVIEW

**Comment by eawest2** [comment]
Ignore the jenkins break, it's a problem with the Jenkins tests, which will be resolved later. If you download, it should work fine.

**Review by anthem123 — APPROVED** [review]
Watched the test work with my own eyes. All good from me.

File: test/kafka/processor-kafka-test.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+describe('Evalutaor Test', () => {
+  it('Evaluates data', async () => {
+    let data = []
+    const evalData = inboundContractData;
+    let outboundJson = outboundContractData;
+      evaluator.evalData(evalData, data);
+      fixTimestamp(data);
+      fixTimestamp(outboundJson);
+
+      should(JSON.stringify(data)).equal(JSON.stringify(outboundJson));
```
Comment (jmfadeley) [inline test/kafka/processor-kafka-test.js:25]: @eawest2 I _really_ don't think it's a good idea to run this test in `processor-kafka-test.js`. The neighboring Kafka test was just a proof of concept to illustrate that connection to Kafka works, and isn't really meant to run in a unit test environment. It actually attempts to connect to Kafka, which is not ideal. 

Your options are A) Delete the 'It tests KafkaJS' test entirely, or B) move 'Evalutaor Test' (misspelled) to another file, change `processor-kafka-test.js` so there's no `-test` suffix, and have `yarn test:kafka` run just your evaluator test instead.

Comment (eawest2) [reply test/kafka/processor-kafka-test.js]: Moved and renamed.

**Review by hmayenve — APPROVED** [review]

File: test/kafka/kafka-integration-test.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ describe('It tests KafkaJS', () => {
     await consumer.disconnect();
     await admin.connect();
     await admin.deleteTopics({
-      topics: [testGroup],
+      topics: [testGroup, "jsonTest"],
```
Comment (jmfadeley) [inline test/kafka/kafka-integration-test.js:35]: This topic, "jsonTest" does not seem to exist when I run it.

**Review by jmfadeley — APPROVED** [review]
I think you should remove "jsonTest" and you're good. I'm getting test failures because the system is using `/r/n` for linebreaks.

**Pull request merged**